mmv1/products/networkconnectivity/ServiceConnectionPolicies.yaml

```diff
@@ -0,0 +1,139 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ServiceConnectionPolicy'
+base_url: 'projects/{{project}}/locations/{{location}}/serviceConnectionPolicies'
+create_url: 'projects/{{project}}/locations/{{location}}/serviceConnectionPolicies?serviceConnectionPolicyId={{name}}'
+self_link: 'projects/{{project}}/locations/{{location}}/serviceConnectionPolicies/{{name}}'
+update_verb: :PATCH
+update_mask: true
+description:
+  'Manage Service Connection Policies.'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/secure-web-proxy/docs/reference/networkconnectivity/rest/v1/projects.locations.networkConnectionPolicies'
+  guides:
+    'About Service Connection Policies': 'https://cloud.google.com/vpc/docs/about-service-connection-policies#service-policies'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+import_format:
+  ['projects/{{project}}/locations/{{location}}/serviceConnectionPolicies/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_policy_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-network-connectivity-policy'
+      consumer_network_name: 'consumer-net'
+      consumer_subnetwork_name: 'consumer-subnet'
+      producer_network_name: 'producer-net'
+      producer_subnetwork1_name: 'producer-subnet1'
+      producer_subnetwork2_name: 'producer-subnet2'
+      service_class_name: 'my-basic-service-class'
+    test_vars_overrides:
+      service_class_name: 'envvar.GetTestServiceClassFromEnv(t)'
```

Comment:
I spoke with the engineer internally, and this can be changed to `service_class_name: 'gcp-memorystore-redis'`. The hardcoded value is fine for all test environments, and so you can also remove the other related code for the environment variable.

Reply:
Great, done that.

Reply:
Would you mind changing just this section back to still use a different value for docs? For example:
```
      service_class_name: 'my-basic-service-class'
    test_vars_overrides:
      service_class_name: 'gcp-memorystore-redis'
```

Reply:
Of course, it makes sense to keep this way.